src/coreclr/src/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -1082,6 +1082,11 @@ internal static float fabsf(float x)
         [RuntimeImport(RuntimeLibrary, "RhpArrayClear")]
         internal static extern bool TryArrayClear(Array array, int index, int length);
 
+#if TARGET_X86 || TARGET_AMD64
+        [DllImport(RuntimeLibrary, ExactSpelling = true, CallingConvention = CallingConvention.Cdecl)]
+        internal static extern unsafe void RhpCpuIdEx(int* cpuInfo, int functionId, int subFunctionId);
```

Comment:
Nit: this should be `RhCpuIdEx` without the `p` if CoreLib calls into it. The `p` suffix would we appropriate if we were to intrinsify it within the compiler.